functions.php

```diff
@@ -7,43 +7,61 @@ function connectDb(string $host, string $user, string $password, string $dbName)
     return $pdo;
 }
 
-function getBottles($pdo): array
-{
+function getBottles($pdo): array {
     $sql = "SELECT * FROM `bottles`";
     $statement = $pdo->prepare($sql);
     $statement->execute();
     $results = $statement->fetchAll();
     return $results;
 }
 
-function addBottle(object $pdo, string $itemName, string $purchaseLocation,string $type, string $purchaseDate)
-{
+function addBottle(PDO $pdo, string $itemName, string $purchaseLocation,string $type, string $purchaseDate) {
     $sql = "INSERT INTO `bottles`(itemname, purchaselocation, type, purchasedate) VALUES (?, ?, ?, ?)";
     $stmt = $pdo->prepare($sql);
     $stmt->execute([$itemName, $purchaseLocation, $type, $purchaseDate]);
 }
 
+function validateDate(string $date, string $format = 'Y-m-d'): bool
+{
+    $dateObject = DateTime::createFromFormat($format, $date);
+    return $dateObject && $dateObject->format($format) === $date;
+}
+
+function checkFormSubmission(array $postArray): array {
+    if(empty($postArray['item-name']) ||  empty($postArray['purchase-location']) || empty($postArray['type']) || empty($postArray['purchase-date'])){
+        return [false, "Please enter a value for all fields."];
+    }
+    if(!validateDate($postArray['purchase-date'])){
+        return [false, "Please ensure you enter a date in the format yyyy-mm-dd."];
+    }
+    $alcoholTypes = ["Rum", "Rye", "Vodka", "Whisky"];
+    if(!in_array($postArray['type'], $alcoholTypes)) {
+        return [false, "You must choose one of the options for alcohol type."];
+    }
```

Comment:
this should be in a separate function - separation of concerns - 1 function should only have 1 responsibility